routes/web.php

```diff
@@ -42,3 +42,4 @@
 
 // Images
 $router->post('images/{postId}', 'Legacy\Web\ImagesController@update');
+$router->get('originals/{post}', 'Legacy\Web\OriginalsController@show');
```

Comment:
Is there a reason we want this new route to be in `Legacy`?

Reply:
We really don't! 😩 I need to move these into just a plain ol' `Web/` namespace...

Reply:
Gotcha, sounds out of scope for this PR!